src/fiatExchanges/ReviewScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import { CICOFlow } from './utils'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectReview>
+
+export default function FiatConnectReviewScreen({ route, navigation }: Props) {
+  const { t } = useTranslation()
+
+  // TODO(satish): get all hardcoded fields from props
+  const { flow } = route.params
+
+  return (
+    <SafeAreaView style={styles.content}>
+      <View>
+        <View style={styles.amountContainer}>
+          <Text style={styles.amountText}>20.01 cUSD</Text>
+        </View>
+        <TransactionDetails flow={flow} />
+        <PaymentMethod />
+      </View>
+      <Button
+        style={styles.submitBtn}
+        type={BtnTypes.PRIMARY}
+        size={BtnSizes.FULL}
+        text={
+          flow === CICOFlow.CashIn
+            ? t('fiatExchangeFlow.cashIn.fiatConnectReviewButton')
+            : t('fiatExchangeFlow.cashOut.fiatConnectReviewButton')
+        }
+        onPress={() => {}}
+      />
+    </SafeAreaView>
+  )
+}
+
+function TransactionDetails({ flow }: { flow: CICOFlow }) {
+  const { t } = useTranslation()
+  return (
+    <View style={styles.sectionContainer}>
+      <Text style={styles.sectionHeaderText}>
+        {t('fiatConnectReviewScreen.transactionDetails')}
+      </Text>
+      <LineItemRow
+        style={styles.sectionMainTextContainer}
+        textStyle={styles.sectionMainText}
+        title={
+          flow === CICOFlow.CashIn
+            ? t('fiatExchangeFlow.cashIn.transactionDetailsAmount')
+            : t('fiatExchangeFlow.cashOut.transactionDetailsAmount')
+        }
+        amount="$20.00"
+      />
+      <LineItemRow
+        title="25 cUSD @ $1.00"
+        amount="20.0010 cUSD"
+        style={styles.sectionSubTextContainer}
+        textStyle={styles.sectionSubText}
+      />
+      <LineItemRow
+        title={t('feeEstimate')}
+        amount="$3.99"
+        style={styles.sectionSubTextContainer}
+        textStyle={styles.sectionSubText}
+      />
+    </View>
+  )
+}
+
+function PaymentMethod() {
+  const { t } = useTranslation()
+  return (
+    <View style={styles.sectionContainer}>
+      <Text style={styles.sectionHeaderText}>{t('fiatConnectReviewScreen.paymentMethod')}</Text>
+      <View style={styles.paymentMethodContainer}>
+        <View style={styles.paymentMethodDetails}>
+          <View style={styles.sectionMainTextContainer}>
+            <Text style={styles.sectionMainText}>Visa (...4666)</Text>
+          </View>
+          <View style={styles.sectionSubTextContainer}>
+            <Text style={styles.sectionSubText}>Via OzzyPay | Est 3-10 business days</Text>
+          </View>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image
+            source={{
+              uri:
+                'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Framp.png?alt=media',
+            }}
+            style={styles.paymentImage}
+            resizeMode="center"
+          />
```

Comment:
Oh, for this image
1. Use the wide logo, all the photos we use will be the wide ones
2. Use the exact same styles/settings as https://github.com/valora-inc/wallet/blob/main/src/fiatExchanges/PaymentMethodSection.tsx#L137-L144

Reply:
Chatted with Nitya, looks like we wanted to use a bank account logo specific to the fiat account instead of the provider logo. Since this is hard to get, we can skip for the MVP, but I just left the code in place with an optional prop for image. Can also clean this up and add later? thoughts?